library/src/main/java/org/apache/apex/malhar/lib/window/impl/SpillableWindowedPlainStorage.java

```diff
@@ -143,4 +149,16 @@ public void setup(Context.OperatorContext context)
   public void teardown()
   {
   }
+
+  @Override
+  public void purge(long horizonMillis)
+  {
+    SpillableStateStore store = scc.getStore();
+    if (store instanceof ManagedTimeUnifiedStateImpl) {
```

Comment:
It will not be purged. I think I should make SpillableStateStore extend ManagedTimeUnifiedStateImpl.